### PR TITLE
Fix XML code blocks

### PR DESCRIPTION
**Pull request by nxtn**
Fix XML attributes.

See #9744

### CODE REVIEW

**Comment by Thraka** [comment]
@NextTurn I reviewed this and it looks good. But something else happened that is causing a conflict.

File: docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md [hunk 1]
```diff
@@ -19,13 +19,12 @@ Configures the Hypertext Transfer Protocol (HTTP) proxy server.
...
   
 ```xml  
-      <defaultProxy  
-        enabled="true|false"  
-        useDefaultCredentials="true|false">  
-           <bypasslist> … </bypasslist>  
-           <proxy> … </proxy>  
-           <module> … </module>  
-      </defaultProxy>
+<defaultProxy enabled="true|false"
```
Comment (karelz) [inline docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md]: Can you please use original indentation (4 spaces for subelements and 2 spaces for attributes) and line wrapping? (multi-line attributes start on new line) I assume it is consistent with other docs.

(applies to multiple occurrences later on as well)

File: docs/framework/network-programming/enabling-and-disabling-ipv6.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ To use the IPv6 protocol, ensure that you are running a version of the operating
  The .NET Framework has a configuration switch set in a configuration file as follows  
   
 ```xml  
-<system.net>…  
-    <settings>…  
-        <ipv6 enabled="true"/>…  
-    </settings>…  
+<system.net>...
```
Comment (karelz) [inline docs/framework/network-programming/enabling-and-disabling-ipv6.md]: Nit: I would put each `...` on new line, unless it is in the form `<elem>...</elem>`.

File: docs/framework/wcf/samples/message-security-sample.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ This sample demonstrates how to implement an application that uses the `basicHtt
                bindingConfiguration="Binding1"   
                contract="Microsoft.ServiceModel.Samples.ICalculator" />  
     </service>  
-  </services>   …  
+  </services>...
```
Comment (karelz) [inline docs/framework/wcf/samples/message-security-sample.md]: `...` on new line, it can be easily misses behind the closing elem tag.

**Review by karelz — COMMENTED** [review]
Nice changes! You found quitre a few mistakes, thanks!
Couple of minor comments above.

**Comment by karelz** [comment]
BTW: I would suggest to stop adding additional changes which are not part of feedback. Otherwise the review will become messy ...
You can submit another PR with additional mistakes you find ...

File: docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md (hunk 1 above)
Comment (NextTurn) [reply docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md]: On the contrary, I think my changed styles are consistent with other docs.

See also #9852

**Comment by NextTurn** [comment]
I'm sorry that I haven't seen your comments at that moment ...

File: docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md (hunk 1 above)
Comment (NextTurn) [reply docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md]: According to the default behaviors of `XmlWriterSettings.NewLineOnAttributes`, it could be better to use 2 spaces indentation for both subelements and attributes, and start a new line for all attributes if there are any multi-line ones.

File: docs/framework/configure-apps/file-schema/wcf/httpstransport.md [hunk 4]
```diff
@@ -24,12 +24,11 @@ Specifies an HTTP transport for transmitting SOAP messages for a custom binding.
                 maxBufferSize="Integer"
                 maxReceivedMessageSize="Integer"
                 proxyAddress="Uri"
-                proxyAuthenticationScheme="None/Digest/Negotiate/Ntlm/Basic/Anonymous"
+                proxyAuthenticationScheme="None/Digest/Negotiate/Ntlm/Basic/Anonymous/IntegratedWindowsAuthentication"
```
[the remark this answers is not in the document]
Comment (NextTurn) [reply docs/framework/configure-apps/file-schema/wcf/httpstransport.md]: https://github.com/dotnet/docs/blob/master/docs/framework/configure-apps/file-schema/wcf/httpstransport.md#attributes-and-elements

File: docs/framework/configure-apps/file-schema/wcf/endpoint-element.md [hunk 5]
```diff
@@ -72,8 +72,8 @@ Specifies binding, contract, and address properties for a service endpoint, whic
           binding="customBinding"
           contract="HelloWorld">
   <headers>
-    <region xmlns="http://tempuri.org/">EastCoast</region>
-    <member xmlns="http://tempuri.org/">Gold</member>
+    <Region xmlns="http://tempuri.org/">EastCoast</Region>
```
[the remark this answers is not in the document]
Comment (karelz) [reply docs/framework/configure-apps/file-schema/wcf/endpoint-element.md]: Did you confirm it is not a bug in the table that it is capitalized there?

[the remark this answers is not in the document]
Comment (NextTurn) [reply docs/framework/configure-apps/file-schema/wcf/endpoint-element.md]: They were originally capitalized and incorrectly changed in #9852. I've reviewed this PR and reverted these changes.

[the remark this answers is not in the document]
Comment (karelz) [reply docs/framework/configure-apps/file-schema/wcf/endpoint-element.md]: Just the fact they were capitalized before does not mean it was the right value ... we should look at how they are defined in code / XML schema IMO.

File: docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md (hunk 1 above)
Comment (NextTurn) [reply docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md]: I would expect a consensus since there are unformatted blocks elsewhere that need to be fixed.

Comment (karelz) [reply docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md]: That should be separate discussion IMO.

File: docs/framework/configure-apps/file-schema/wcf/endpoint-element.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/framework/configure-apps/file-schema/wcf/endpoint-element.md]: @Lxiamail @zhenlan where can we check if this is correct?
In VS, I see totally different elements under \<headers>.

[the remark this answers is not in the document]
Comment (Thraka) [reply docs/framework/configure-apps/file-schema/wcf/endpoint-element.md]: I think headers is an open node, you define what you want to be included in the http request/response headers?

[the remark this answers is not in the document]
Comment (mconnew) [reply docs/framework/configure-apps/file-schema/wcf/endpoint-element.md]: The problem here is the Child Elements table shouldn't be there. From the documentation text it says:

>  Entries in the endpoint header collection are user-defined XML elements. Each element has to be well-formed XML.

The member and region elements were just examples of how you can add a custom header, the names can be arbitrary as long as it's valid xml. This specific change should be reverted and instead remove the child elements table.

File: docs/framework/configure-apps/file-schema/wcf/headers.md [hunk 6]
```diff
@@ -14,8 +14,8 @@ An endpoint can be addressed by one or more SOAP headers in addition to its basi
   
 ```xml  
 <headers>
-  <region xmlns="Uri">"String"</region>
-  <member xmlns="Uri">"String"</member>
+  <Region xmlns="Uri">"String"</Region>
+  <Member xmlns="Uri">"String"</Member>
```
Comment (mconnew) [inline docs/framework/configure-apps/file-schema/wcf/headers.md]: Same fix here. Revert this change and instead remove the Child Elements table.

File: docs/framework/configure-apps/file-schema/wcf/httpstransport.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mconnew) [reply docs/framework/configure-apps/file-schema/wcf/httpstransport.md]: I'm pretty certain this is incorrect. If you look at [this code ](https://referencesource.microsoft.com/#System.ServiceModel/System/ServiceModel/HttpProxyCredentialType.cs,29)you can see that it will throw an exception if anything other than Anonymous, Basic, Digest, Ntlm or Negotiate is specified.

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 7]
```diff
@@ -282,8 +282,8 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
 <wst:RequestSecurityTokenResponse   
 xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd"   
 xmlns:wssu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-     wssecurity-utility-1.0.xsd"
```
Comment (mconnew) [inline docs/framework/wcf/feature-details/transaction-protocols.md]: The namespace definition here is also wrong. There's whitespace in the middle of the uri

File: docs/framework/configure-apps/file-schema/wcf/httpstransport.md (hunk 4 above)
Comment (mconnew) [inline docs/framework/configure-apps/file-schema/wcf/httpstransport.md]: Commenting again as previous conversation was marked as resolved.

I'm pretty certain this is incorrect. If you look at [this code](https://referencesource.microsoft.com/#System.ServiceModel/System/ServiceModel/HttpProxyCredentialType.cs,29) you can see that it will throw an exception if anything other than Anonymous, Basic, Digest, Ntlm or Negotiate is specified.

Comment (NextTurn) [reply docs/framework/configure-apps/file-schema/wcf/httpstransport.md]: Is the table also incorrect?

File: docs/framework/configure-apps/file-schema/wcf/headers.md (hunk 6 above)
Comment (NextTurn) [reply docs/framework/configure-apps/file-schema/wcf/headers.md]: Is it OK to remove the Child Elements section?

File: docs/framework/configure-apps/file-schema/wcf/httpstransport.md (hunk 4 above)
Comment (mconnew) [reply docs/framework/configure-apps/file-schema/wcf/httpstransport.md]: I believe so. I'll need to write some test code to verify in case I've missed something. I'll update this conversation once I have done so.

Comment (mconnew) [reply docs/framework/configure-apps/file-schema/wcf/httpstransport.md]: I just verified and we throw an exception if `IntegratedWindowsAuthentication` is specified. This is because it's value is actually `Negotiate | Ntlm` and we only allow a single authentication scheme to be specified. So it does need to be removed from the table. It might be worth adding a note after the where it says this is of type AuthenticationSchemes that this value isn't supported.

**Review by mconnew — APPROVED** [review]
All the WCF and serialization changes look good. Thanks for cleaning these up.

File: docs/framework/wcf/feature-details/using-the-nethttpbinding.md [hunk 8]
```diff
@@ -39,7 +39,7 @@ ms.assetid: fe134acf-ceca-49de-84a9-05a37e3841f1
         </binding>  
       </netHttpBinding>  
     </bindings>  
-    <!- ... -->   
+    <!-- ... -->
```
Comment (Thraka) [inline docs/framework/wcf/feature-details/using-the-nethttpbinding.md]: You're using ... in other places without the XML comment, so I would do the same here.

**Comment by Thraka** [comment]
Final two things to fix and then I think this is good!

**Review by Thraka — CHANGES_REQUESTED** [review]
Final two things to fix and then I think this is good!

**Review by Thraka — APPROVED** [review]
